administration/src/i18n.ts

```diff
@@ -32,6 +32,7 @@ export const loadTranslations = (): Translations => ({
     stores: deTranslations.stores,
     userSettings: deTranslations.userSettings,
     users: deTranslations.users,
+    component_Accordion: deTranslations.component_Accordion,
```

Comment:
💭 hm currently we set namespaces for pages and we already, so i would tend to say this can be in `applicationOverview` 
If we really want to have a namespace for "base component" that we reuse in several places, we can even create a "baseComponents" namespace or put it in misc. 
I don't see an advantage in creating a namespace for each component.
What was your intend?

Reply:
Yes, I had the idea of one namespace per component for reusable components. I would like to refactor the file layout later after the MUI migration and wanted to introduce a `/shared` dir for components and other functionality at some point. 

We could use one namespace for all modules in (future) `/shared`, but I still would prefix component specific strings with e.g. `Accordion-` or something. Would you agree?